### ISSUE TITLE
Anyone want to help maintain VSTOContrib?

### ISSUE THREAD

**Issue by JakeGinnivan** [JakeGinnivan/VSTOContrib#60]
I am just not working in the office space and haven't been for a while. I am struggling to get back to this project. 

I would be happy to help anyone get up to speed on this project and try and take over maintenance of it. In the future I might get back to this but at the moment I just don't know. 

Sorry all using this, I really enjoyed this project and I had grander plans but there are too many other open source projects I am working on.

**Comment by umerfaruk** [comment]
Hi Jake. I will be happy to give you a hand with this, if you like.

**Comment by mmfoscarpeerio** [comment]
I've recently used this project and it covers a lot of enterprise practices perfectly and very well implemented. If you still have a tight schedule to support it, I'll be glad to coordinate something and consider keeping the maintenance as you suggested.

**Comment by shahshyam** [comment]
Hi, Can contribute you for this project. I have experience in VSTO and COM Addin.

**Comment by JakeGinnivan** [comment]
If you are interested I can add you as maintainers :) Did you want me to do that?

**Comment by shahshyam** [comment]
yes: )

**Comment by Quadrature** [comment]
Is anyone maintaining this project? I have a few questions...

**Comment by shahshyam** [comment]
Ok share please it?

**Comment by advanced373** [comment]
Hello! Is this still maintained?

**Comment by shahshyam** [comment]
@advanced373 If you need any help ? please let me know

**Comment by advanced373** [comment]
No, I'm just interested in contribution because I'm working with those kinds of stuff and I gathered some experience.
